asm/macros/movement.inc

```diff
@@ -1,131 +1,166 @@
-	.macro create_movement_action name
-	enum _\name
+	.macro create_movement_action name, value
 	.macro \name
```

Comment:
These should probably just be constants instead of macros, I think?  The macros simply output a single byte.

Reply:
The macros are simplified names for the constants (plus the byte directive). Movement scripts would be unwieldy with the C defines

Reply:
True, I forgot about that.